index.rst

```diff
@@ -352,28 +385,145 @@ To make sure the system would work properly we had to enable two additional feat
+It is important to note that the :ref:`configuration <Appendix-Final-Shared-Memory-OpenSplice-Configuration>` discussed in :ref:`Increasing-OpenSplice-daemon-resources` affect only the daemons and not those processes that will still rely on single process mode.
+In order to allow these processes to still run reliably, and also in an attempt to require less resources from the daemons, a new DDS data partitioning schema was implemented.
+
+As explained in :ref:`DDS-Domain-and-DDS-Partition` this feature allows the DDS domain to filter out topics for readers that join the domain in a particular partition.
+To take advantage of this feature without increasing the scope of system configuration complexity, we decided to adopt two different partitions for each component of the system; one partition exclusively for commands and a second one for telemetry, events and acknowledgements.
+This configuration allows us to maximize the segmentation of the system while minimizing the computing resources required by components.
+
+The motivation for this partitioning scheme derives mainly from the system architecture.
+The system architecture specify mainly two types components; Controllers and Remotes, in SalObj nomenclature.
+Controllers are the base components upon which CSCs are created.
+They mainly consist of applications that read commands and write telemetry, events and acknowledgements.
+At the same time, to communicate with a CSC one would use a Remote, which write commands and read telemetry, events and acknowledgements.
+
+By allowing a separate partition for read and write operations, components can share most of the resources needed to communicate in both directions while still minimizing data duplication (as shown in :numref:`fig-partition`).
+One could argue that there could be a partition for each individual topic of the system, which would allow readers to only receive the data they are actually requesting.
+The reason for avoiding such thin partitioning is that it would require substantial resource duplication.
+For instance, DDS allows applications to share substantial resources when creating readers and writers on the same partition.
+At the same time, the software architecture allows us to safely share most of those resources for an individual component so, splitting the topics for a component into more partitions means there would be less shared resources.
+
+Details on how this is implemented can be found in `SalObj documentation <https://ts-salobj.lsst.io>`__.
+
+.. _Quality-of-Service-configuration:
+
+Quality of Service configuration
+--------------------------------
+
+The Quality of Service (QoS) are topic specify configurations that control many aspects of how information is handled by DDS, including how reliable messages are exchanged and if a topic has historical data (for later joiners).
+Our architecture apply QoS configuration based on the message type, e.g., commands, events, telemetry and acknowledgements.
```

Comment:
"apply QoS" -> "specifies QoS" (or maybe "applies QoS" but I think "specifies" is better).

Reply:
"acknowledgements" -> "command acknowledgements"